recipes-core/systemd/systemd-conf_%.bbappend

```diff
@@ -19,3 +19,7 @@ do_install:append() {
 
 	sed -i "s/@@MACHINE@@/${MACHINE}/g" ${D}${systemd_unitdir}/system.conf.d/10-${BPN}.conf
 }
+
+do_install:append:ti-soc() {
+	sed -i '$ d' ${D}${systemd_unitdir}/system.conf.d/10-${BPN}.conf
```

Comment:
That's pretty brittle; how about:

```
sed -i '/RuntimeWatchdogSec=/d' ${D}${systemd_unitdir}/system.conf.d/10-${BPN}.conf
```

Reply:
Thanks, I've already changed the command.